test/ParseableInterface/synthesized.swift

```diff
@@ -11,13 +11,21 @@ public enum HasRawValue: Int {
   // CHECK-DAG: }
 } // CHECK: {{^}$}}
 
-// CHECK-LABEL: @objc public enum ObjCEnum : Int {
+#if os(Windows) && (arch(arm64) || arch(x86_64))
+@objc public enum ObjCEnum: Int32 {
```

Comment:
Oh, just change this to Int32 everywhere, then. The Int-ness isn't important.

Reply:
I like simpler!